orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/model/Execution.groovy

```diff
@@ -120,4 +121,21 @@ abstract class Execution<T> implements Serializable {
       return Optional.empty()
     }
   }
+
+  static class PausedDetails implements Serializable {
+    String pausedBy
+    String resumedBy
+
+    Long pauseTime
+    Long resumeTime
+
+    boolean isPaused() {
```

Comment:
I wasn't sure because `isXXX` should get picked up as a Java Bean property, I think.